config.json

```diff
@@ -149,5 +149,9 @@
     },
     "integrityChecks": {
         "objectPutRetention": true
+    },
+    "serverAccessLogs": {
+        "enabled": false,
```

Comment:
Could we have two types of "enabled"?
1. "Are we writing the `server-access.log`" (default `false` for Artesca). We could call this field `"generateAcessLogFile"`
2. "Is the feature enabled in Federation" (`false` by default, will be configured by Federation). When this is `false`: (1) The API returns `not implemented`, and (2) Cloudserver always sets `loggingEnabled` to `false`. That way, we can disable the feature in Federation.